pkg/reconciler/namespace/namespace.go

```diff
@@ -211,17 +222,22 @@ func (c *Controller) isValidCluster(ctx context.Context, lclusterName, clusterNa
 	if err != nil {
 		return false, err
 	}
-	isReady := conditions.IsTrue(cluster, clusterv1alpha1.ClusterReadyCondition)
-	if !isReady {
+	if !conditions.IsTrue(cluster, clusterv1alpha1.ClusterReadyCondition) {
```

Comment:
(No action required) Consider retaining the use of `isReady` even if it is not needed for return. 'Named conditionals' tend to improve code readability.